lib/spec/swagger/utils.js

```diff
@@ -164,6 +164,11 @@ function replaceUnsupported (jsonSchema) {
     if (jsonSchema.patternProperties) {
       jsonSchema.additionalProperties = { type: 'string' }
       delete jsonSchema.patternProperties
+    } else if (jsonSchema.const) {
+      // OAS 3.1 supports `const` but it is not supported by `swagger-ui`
+      // https://swagger.io/docs/specification/data-models/keywords/
```

Comment:
The old comment, here in the `swagger` directory was fine.

- Swagger is an alias for OAS version 2 that does not support `const`
- The other comment, in `openapi` folder, is the file to support OAS version 3. So the comment there must be different from the following

```suggestion
      // Handle const, that is not part of OpenAPI definitions
```

Reply:
thanks, fixed